tests/functional/s/slots_checks.py

```diff
@@ -63,6 +63,13 @@ class NinthBad(object):
 class TenthBad(object):
     __slots__ = [1 + 2 + 3]  # [invalid-slots-object]
 
+class EleventhBad:  # [invalid-slots]
+    """type-annotated __slots__ with no value"""
+    __slots__: str
```

Comment:
This is superfluous, but not invalid, right? Is there a good reason to emit `invalid-slots`?

Reply:
Good point @jacobtylerwalls!
Perhaps separately we can deal with this if we want to. In the meantime we can ignore this & focus on the crash aspect itself?